src/Traits/HasRoles.php

```diff
@@ -381,7 +381,9 @@ public function getRoleNames(): Collection
 
     protected function getStoredRole($role): Role
     {
-        if (is_string($role)) {
+        if (is_numeric($role)) {
+            return app(Role::class)->findById($role, $this->getDefaultGuardName());
+        } elseif (is_string($role)) {
```

Comment:
This can be changed to a single `if` statement as the previous statement will return if true.

Reply:
Like this, perhaps?

    return is_numeric($role)
            ? app(Role::class)->findById($role, $this->getDefaultGuardName())
            : app(Role::class)->findByName($role, $this->getDefaultGuardName());

Reply:
You could do that, but I think @AlexVanderbist meant to just remove the `elseif` clause, like this:

```php
if (is_numeric($role)) {
   return foo;
}
return bar;
```